gfw_pixetl/decorators.py

```diff
@@ -4,6 +4,10 @@
 from multiprocessing import Process, Queue
 
 
+class SubprocessKilledError(Exception):
+    pass
+
```

Comment:
move to errors.py

Reply:
I had it there, but it caused a circular import. :( Will try to resolve.